ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Your title above should be a short description of what
+is the issue. Do not include the pull request number in the title. -->
```

Comment:
Since this is an issue there shouldn't be a pull request number yet and since they didn't file the issue yet they won't know the number...so perhaps that sentence can be removed.